### PR TITLE
docs: Improve upgrade/downgrade documentation

### PR DESCRIPTION
**Pull request by Ebaneck**
**Component**:

'docs'

**Context**: 

see #2076 

**Summary**:

Make upgrade/downgrade docs consistent

**Acceptance criteria**: 

---

Closes: #2076

### CODE REVIEW

File: docs/operation/downgrade.rst [hunk 1]
```diff
@@ -1,9 +1,50 @@
 Downgrade Guide
 ***************
-This section describes the logical steps for downgrading **MetalK8s**.
+Downgrading a MetalK8s cluster is handled via utility scripts which are
+packaged with your current installation.
+This section describes a reliable downgrade procedure for **MetalK8s**
+including all the components that make up the stack.
```
Comment (lucieleonard) [inline docs/operation/downgrade.rst]: make up== are included?

File: docs/operation/downgrade.rst [hunk 2]
```diff
@@ -1,9 +1,50 @@
...
+******************
+.. note::
+
+    MetalK8 supports downgrade **strictly** from one supported
+    minor version to another. For example:
+
+    - Downgrade from 2.1.x to 2.0.x
+    - Downgrade from 2.2.x to 2.1.x
+
+    Please refer to the release notes for more information.
```
Comment (lucieleonard) [inline docs/operation/downgrade.rst]: Will need to provide a link to the release notes. or at least explain where one can find them

File: docs/operation/downgrade.rst [hunk 3]
```diff
@@ -1,9 +1,50 @@
...
+
+Downgrade Pre-requisites
+************************
+Prior to beginning the downgrade step listed below, make sure to complete the
```
Comment (lucieleonard) [inline docs/operation/downgrade.rst]: Before proceeding with the downgrade procedure, make sure..

File: docs/operation/downgrade.rst [hunk 4]
```diff
@@ -1,9 +1,50 @@
...
+pre-requisites listed in :doc:`/operation/preparation`.
+
+To try out the feasibility of your downgrade demands, run the following
+command.
```
Comment (lucieleonard) [inline docs/operation/downgrade.rst:28]: Not sure I understand here.
Is it to test if the downgrade will succeed?
If so, change it to "You can test if your environment will successfully downgrade with the following command."

File: docs/operation/upgrade.rst [hunk 5]
```diff
@@ -17,17 +19,33 @@ Supported Versions
 
 Upgrade Pre-requisites
 **********************
-Prior to beginning the upgrade steps listed below, make sure to complete the
+Prior to beginning the upgrade step listed below, make sure to complete the
 pre-requisites listed in :doc:`/operation/preparation`.
 
+To try out the feasibility of your upgrade demands, run the following command.
```
Comment (lucieleonard) [inline docs/operation/upgrade.rst]: same as for downgrade

File: docs/operation/upgrade.rst [hunk 6]
```diff
@@ -17,17 +19,33 @@ Supported Versions
...
+To upgrade a MetalK8s cluster, run the utility script shipped
+with the **new** version you want to upgrade to providing it with the
+destination version:
 
 - From the :term:`Bootstrap node`, launch the upgrade.
 
+.. important::
+
+    The version prefix metalk8s-**X.X.X** as used below during a MetalK8s
+    upgrade must be the lastest available version you would like to upgrade to.
```
Comment (lucieleonard) [inline docs/operation/upgrade.rst]: must be the latest available (stable?) version you wish to upgrade to.

Comment (Ebaneck) [reply docs/operation/upgrade.rst]: would prefer to stick to ```lastest available or new```  for the following reasons.
To run upgrades
- We somehow add the ISO images of the two versions in the cluster hence the phrase lastest available.
That is if i want to upgrade from 2.3 to 2.4, then these two ISO images need to be present in the cluster.

Comment (Ebaneck) [reply docs/operation/upgrade.rst]: would rather use ```new```

**Review by lucieleonard — APPROVED** [review]

**Pull request merged**